hr_citizenship/views/hr_applicant.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_form_applicant_citizenship" model="ir.ui.view">
+        <field name="name">applicant citizenship</field>
+        <field name="model">hr.applicant</field>
+        <field name="inherit_id" ref="hr_recruitment.crm_case_form_view_job"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='type_id']" position="after">
+                <field name="citizenship_ids" readonly="[('empl_id', '!=', False)]"
```

Comment:
are you sure these domains are computed? AFAIK you have to use `attrs`.

Reply:
yes you are right, took it from [sale_wizard](https://github.com/odoo/odoo/blob/49ca43d75cb9a97642c820c2466d454f1ce604cb/addons/sale/wizard/sale_make_invoice_advance_views.xml#L13) it probably or broken or work on simple integer field, I'll check later